### DISCUSSION TITLE
multiple immich_microservices possible?

### DISCUSSION THREAD

**Discussion by tippfehlr** [immich-app/immich#7043] [Q&A] [answered]
In short, title. Is running multiple containers of immich_microservices (each pointed to the same redis/postgres) possible?

For example, I would additionally start it on my (much more powerful) computer too. Do I need to stop the existing immich_microservices?

**Comment by bo0tzz** [comment, marked as answer]
Yes, all the immich containers should allow for running multiple instances without issue.
